pkg/provider/azure_loadbalancer_accesscontrol.go

```diff
@@ -106,3 +106,53 @@ func (az *Cloud) listSharedIPPortMapping(
 
 	return rv, nil
 }
+
+func (az *Cloud) listAvailableSecurityGroupDestinations(_ context.Context) ([]netip.Addr, error) {
+	services, err := az.serviceLister.List(labels.Everything())
+	if err != nil {
+		return nil, fmt.Errorf("list all services: %w", err)
+	}
+
+	nodes, err := az.nodeLister.List(labels.Everything())
```

Comment:
unmanaged nodes should always be excluded, let's skip them here

Reply:
which label should be used to filter managed nodes?

Reply:
here are the labels https://cloud-provider-azure.sigs.k8s.io/topics/cross-resource-group-nodes/#unmanaged-nodes

Reply:
Nodes with `kubernetes.azure.com/managed=false`  or `node.kubernetes.io/exclude-from-external-load-balancers=true` should be excluded

Reply:
https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/pkg/provider/azure.go#L109